mkosi.profiles/splash/mkosi.conf.d/fedora/mkosi.conf

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=fedora
+
+[Content]
+Packages=
+        plymouth-theme-spinner
+InitrdPackages=
+        plymouth-theme-spinner
```

Comment:
Are there equivalent packages for other distros? If yes then we should likely make this profile work for others distros, too. (I also would prefer bgrt as the default theme)

Reply:
This isn't needed on arch because it's already included in the main plymouth package. And yeah bgrt is the default on Fedora, but the actual theme is in this package (alongside the spinner one).

Reply:
> This isn't needed on arch because it's already included in the main plymouth package.

Ah okay

> And yeah bgrt is the default on Fedora, but the actual theme is in this package (alongside the spinner one).

Weird choice by them, didn't knew that either.

What about the other distros? Debian/Ubuntu, openSUSE, etc?

Reply:
ParticleOS only supports Arch and Fedora at the moment. Happy to expand this to cover other distros if support for them is added down the road, but at this stage it seems premature.